### DISCUSSION TITLE
force refresh page can see  flash of unstyled content with antd

### DISCUSSION THREAD

**Discussion by XShellv** [vercel/next.js#16654] [Help]
# Bug report
force refresh page can see  flash of unstyled content

## Describe the bug
press ctrl+shift+R forcing refresh the production page can see  flash of unstyled content , at the same time the antd menu option disappear , I really do not know whether it is antd bug or next bug ????? I have try to import the css by `link` tag , but it does not work.

## Expected behavior
expect no flash of unstyled content

## Screenshots
before:
![91581593-30fa8980-e981-11ea-88f5-9f9e38a7f99e](https://user-images.githubusercontent.com/45475139/91582421-5f2c9900-e982-11ea-9e6e-52d8272711cb.png)
after:
<img width="871" alt="捕获" src="https://user-images.githubusercontent.com/45475139/91582493-7b303a80-e982-11ea-86c0-0f32aff03b93.PNG">

## System information

- OS: Win64
- Browser  chrome, safari
- Version of Next.js: v9.5.2
- Version of Node.js: v12.3.0

**Comment by awareness481** [comment]
Any chance we could have a repo with a reproduction of this issue? 

Next.js has a lot of tests internally to ensure FOUC doesn't happen so this issue is very interesting.

**Comment by XShellv** [comment]
> Any chance we could have a repo with a reproduction of this issue?
> 
> Next.js has a lot of tests internally to ensure FOUC doesn't happen so this issue is very interesting.

May be I import style incorrectly , thank you for your answer !😄

**Comment by Nou-r** [comment]
Hey how did you solve this? I'm getting the exact same issue!

Thanks,
Nour
